+++
title = "Dynamics of mitochondrial heteroplasmy in three families investigated via a repeatable re-sequencing study"
date = "2011-06-23T00:00:00"
draft = false

authors = ["H Goto", "B Dickins", "E Afgan", "[__IM Paul__](https://pennstate.pure.elsevier.com/en/persons/ian-paul)", "J Taylor", "[__KD Makova__](http://www.bx.psu.edu/makova_lab)", "A Nekrutenko"]

publication = "_Genome Biology_"
publication_short = "_GENOME BIOL_"
publication_types = ["2"]

abstract = "### Background\nOriginally believed to be a rare phenomenon, heteroplasmy - the presence of more than one mitochondrial DNA (mtDNA) variant within a cell, tissue, or individual - is emerging as an important component of eukaryotic genetic diversity. Heteroplasmies can be used as genetic markers in applications ranging from forensics to cancer diagnostics. Yet the frequency of heteroplasmic alleles may vary from generation to generation due to the bottleneck occurring during oogenesis. Therefore, to understand the alterations in allele frequencies at heteroplasmic sites, it is of critical importance to investigate the dynamics of maternal mtDNA transmission.\n### Results\nHere we sequenced, at high coverage, mtDNA from blood and buccal tissues of nine individuals from three families with a total of six maternal transmission events. Using simulations and re-sequencing of clonal DNA, we devised a set of criteria for detecting polymorphic sites in heterogeneous genetic samples that is resistant to the noise originating from massively parallel sequencing technologies. Application of these criteria to nine human mtDNA samples revealed four heteroplasmic sites.\n### Conclusions\nOur results suggest that the incidence of heteroplasmy may be lower than estimated in some other recent re-sequencing studies, and that mtDNA allelic frequencies differ significantly both between tissues of the same individual and between a mother and her offspring. We designed our study in such a way that the complete analysis described here can be repeated by anyone either at our site or directly on the Amazon Cloud. Our computational pipeline can be easily modified to accommodate other applications, such as viral re-sequencing."
abstract_short = ""

image_preview = ""
selected = false
projects = []
tags = []

url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_custom = [{name = "DOI", url = "https://doi.org/10.1186/gb-2011-12-6-r59"}]

math = false
highlight = true

[header]
image = ""
caption = ""
+++
