+++
title = "Reverse Transcription Errors and RNA-DNA Differences at Short Tandem Repeats"
date = "2016-10-01T00:00:00"
draft = false

authors = ["A Fungtammasan", "M Tomaszkiewicz", "R Campos-SÃ¡nchez", "[__KA Eckert__](https://pennstate.pure.elsevier.com/en/persons/kristin-eckert)", "M Degiorgio", "[__KD Makova__](http://www.bx.psu.edu/makova_lab)"]

publication = "_Molecular Biology and Evolution_"
publication_short = "_MOL BIOL EVOL_"
publication_types = ["2"]

abstract = "Transcript variation has important implications for organismal function in health and disease. Most transcriptome studies focus on assessing variation in gene expression levels and isoform representation. Variation at the level of transcript sequence is caused by RNA editing and transcription errors, and leads to nongenetically encoded transcript variants, or RNA-DNA differences (RDDs). Such variation has been understudied, in part because its detection is obscured by reverse transcription (RT) and sequencing errors. It has only been evaluated for intertranscript base substitution differences. Here, we investigated transcript sequence variation for short tandem repeats (STRs). We developed the first maximum-likelihood estimator (MLE) to infer RT error and RDD rates, taking next generation sequencing error rates into account. Using the MLE, we empirically evaluated RT error and RDD rates for STRs in a large-scale DNA and RNA replicated sequencing experiment conducted in a primate species. The RT error rates increased exponentially with STR length and were biased toward expansions. The RDD rates were approximately 1 order of magnitude lower than the RT error rates. The RT error rates estimated with the MLE from a primate data set were concordant with those estimated with an independent method, barcoded RNA sequencing, from a _Caenorhabditis elegans_ data set. Our results have important implications for medical genomics, as STR allelic variation is associated with >40 diseases. STR nonallelic transcript variation can also contribute to disease phenotype. The MLE and empirical rates presented here can be used to evaluate the probability of disease-associated transcripts arising due to RDD."
abstract_short = ""

image_preview = ""
selected = false
projects = []
tags = []

url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_custom = [{name = "DOI", url = "https://doi.org/10.1093/molbev/msw139"}]

math = false
highlight = true

[header]
image = ""
caption = ""
+++
