+++
title = "MAFcounter: an efficient tool for counting the occurrences of k-mers in MAF files"
date = "2025-05-30T00:00:00"
draft = false

authors = ["M Patsakis", "K Provatas", "A Karatzikos", "C Koilakos", "I Mouratidis', "I Georgakopoulos-Soares"]
publication = "_BMC Bioinformatics_"
publication_short = "_BMC Bioinformatics_"
publication_types = ["2"]

abstract = "Motivation: With the rapid expansion of large-scale biological datasets, DNA and protein sequence alignments have become essential for comparative genomics and proteomics. These alignments facilitate the exploration of sequence similarity patterns, providing valuable insights into sequence conservation, evolutionary relationships and for functional analyses. Typically, sequence alignments are stored in formats such as the Multiple Alignment Format (MAF). Counting k-mer occurrences is a crucial task in many computational biology applications, but currently, there is no algorithm designed for k-mer counting in alignment files. Results: We have developed MAFcounter, the first k-mer counter dedicated to alignment files. MAFcounter is multithreaded, fast, and memory efficient, enabling k-mer counting in DNA and protein sequence alignment files with a wide variety of features for k-mer analysis. Availability: MAFcounter is released under GPL license as a suite of binary C++ applications and is available at: https://github.com/Georgakopoulos-Soares-lab/MAFcounter. Keywords: Genomics; Multiple sequence alignment; Proteomics; k-mer counting."

abstract_short = ""

image_preview = ""
selected = false
projects = []
tags = []

url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
url_custom = [{name = "DOI", url = "https://doi.org/10.1186/s12859-025-06172-7"}]

math = false
highlight = true

[header]
image = ""
caption = ""
+++
